src/js/player.js

```diff
@@ -1900,6 +1900,18 @@ class Player extends Component {
   }
 
   /**
+   * Reset the player. Removes all sources, loads the Html tech, and calls `load`.
+   *
+   * @return {Player} Returns the player
+   * @method reset
+   */
+  reset() {
+    this.loadTech_('Html5', null);
```

Comment:
In an offline discussion we decided that using the first item in the techOrder is the best course of action as it matches what happens in videojs if you were to load it in a page without any sources.
See [flash](http://jsfiddle.net/gkatsev/njku477m/4/embedded/result/) and [html](http://jsfiddle.net/gkatsev/njku477m/5/embedded/result/) examples.